source/core/NumericString.cpp

```diff
@@ -1291,6 +1293,38 @@ IntMax ScanIntValues(char formatChar, char* beginPointer, char** endPointer)
     return intValue;
 }
 
+char validTrueValues[5][10] = { "y", "t", "true", "on", "yes" };
+char validFalseValues[5][10] = { "n", "f", "false", "off", "no" };
```

Comment:
Since the size of array of strings and length of each string is unknown, we should initialize like the following:
static const char *validTrueValues[] = { "y", "t", "true", "on", "yes" };
static const char *validFalseValues[] = { "n", "f", "false", "off", "no" };
Making it "static" and "const" implies the intentions also.